src/components/Sidebar/index.tsx

```diff
@@ -0,0 +1,175 @@
+import Drawer from '@mui/material/Drawer';
+import IconButton from '@mui/material/IconButton';
+import List from '@mui/material/List';
+import ListItem from '@mui/material/ListItem';
+import ListItemButton from '@mui/material/ListItemButton';
+import ListItemIcon from '@mui/material/ListItemIcon';
+import ListItemText from '@mui/material/ListItemText';
+
+// Wrapper
+const DrawerWrapper = ({ children, isOpen, isMobile }: { children: ReactNode[], isOpen: boolean, isMobile: boolean }) => {
+  const drawerWidth = isMobile ? '100%' : '300px';
+
+  return (
+    <Drawer
+      sx={{
+        width: isOpen ? drawerWidth : '80px',
+        flexShrink: 0,
+        overflow: 'hidden',
+        transition: '.5s all ease',
+        '& .MuiDrawer-paper': {
+          width: isOpen ? drawerWidth : '80px',
+          padding: '1em',
+          boxSizing: 'border-box',
+          border: 'none',
+          boxShadow: '1px 0px 5px #efefef',
+          transition: '.5s all ease',
+        },
+      }}
+      variant="persistent"
+      anchor="left"
+      open={isMobile ? isOpen : true}
+    >
+      {children}
+    </Drawer>
+  );
+};
+
+// Sidebar Header
+const DrawerHeader = ({ handleDrawerClose, isOpen }: { handleDrawerClose: () => void, isOpen: boolean }) => {
+  return (
+    <>
+      <div style={{
+        display: 'grid',
+        gridTemplateColumns: isOpen ? '1fr 1fr 1fr' : '1fr'
+      }}>
+        {
+          isOpen && (
+            <Fade in={isOpen}>
+              <img
+                alt="WBD Logo"
+                src="logo.png"
+                style={{
+                  height: 40,
+                  gridColumn: '2',
+                  justifySelf: 'center',
+                }}
+              />
+            </Fade>
+          )
+        }
+
+        <IconButton
+          aria-label='close/toggle drawer'
+          onClick={handleDrawerClose}
+          sx={{
+            color: 'primary.main',
+            justifySelf: isOpen ? 'end' : 'center',
+            rotate: isOpen ? '0deg' : '180deg',
+            transition: '.5s all ease',
+          }}>
+          <ChevronLeftIcon />
+        </IconButton>
+      </div>
+      <Divider sx={{
+        backgroundColor: 'primary.main',
+        marginY: '1em',
+        height: '2px'
+      }}
+      />
+    </>
+  );
+};
+
+// Menu
+const DrawerMenu = () => {
+  const menuItems = [
+    {
+      text: 'Página Inicial',
+      icon: <Home />,
+    },
+    {
+      text: 'Cadastro',
+      icon: <AssignmentAdd />,
+    },
+    {
+      text: 'Alunos',
+      icon: <PeopleAlt />,
+    },
+    {
+      text: 'Frequência',
+      icon: <Checklist />,
+    },
+    {
+      text: 'Atividades',
+      icon: <AppRegistration />,
+    },
+  ];
+
+  return (
+    <List
+      sx={{
+        color: 'primary.main',
+      }}
+    >
+      {menuItems.map((item) => (
+        <ListItem key={item.text} disablePadding>
+          <ListItemButton sx={{
+            backgroundColor: '#efefef',
```

Comment:
This color could be added as `background.paper` inside theme on #24 
So as not to have a hardcoded color
```suggestion
            backgroundColor: 'background.paper',
```